Add Spanish translation of the article "Programming ZKPs: From Zero to Hero"

Gelois0 opened the pull request:
**Added the Spanish version of "Programming ZKPs: From Zero to Hero"**

- Adds `content/es/02_programming-zkps-from-zero-to-hero.md`
- Complete translation with all sections, examples, and footnotes
- Maintains the same structure and formatting as the English version

Gelois0 committed:
Create 02_programming-zkps-from-zero-to-hero.md

Gelois0 committed:
Remove English text and fix minor issues

Gelois0 committed:
Update 02_programming-zkps-from-zero-to-hero.md

Gelois0 committed:
fix: Spanish translation of the text about camouflage and collective pseudonyms

Corrected the Spanish translation of the passage explaining why zebras have stripes and the example of humans hiding under a collective name, such as in the Federalist Papers and Bourbaki.

- Revised translation for better clarity and fluency.
- Maintained technical and cultural accuracy.
- Includes original references for tracking.

Gelois0 committed:
fix: Removed duplication in the "Building Our Circuit" section

liminated duplicated content in the translation of the section about compiling the circuit in `example1-solution.circom`. Clarified instructions to run `just build example1` and kept the reference to the official documentation.

oskarth commented on content/es/02_programming-zkps-from-zero-to-hero.md:
```diff
@@ -0,0 +1,752 @@
+tags: ['zero-knowledge']
+draft: false
+layout: PostSimple
+slug: "/es/programming-zkps-from-zero-to-hero"
```
Slug should be without the `/es/` prefix - this is already in the folder layout

See e.g. https://raw.githubusercontent.com/zkintro/zkintro/refs/heads/main/content/zh-tw/01_friendly-introduction-to-zero-knowledge.md

oskarth commented on content/es/02_programming-zkps-from-zero-to-hero.md:
```diff
@@ -0,0 +1,752 @@
+summary: "Aprende a escribir y modificar pruebas de conocimiento cero desde cero. Construirás un esquema de firma digital usando compromisos basados en hash, adquiriendo habilidades prácticas e intuición en programación de ZKP. Al final, tendrás todas las herramientas necesarias para implementar cosas como firmas grupales."
+---
+
+
```
Can add a short italics line like this for translator:

<img width="533" height="101" alt="image" src="https://github.com/user-attachments/assets/fbb13d21-02b3-4b07-abdb-4d4036e00c2f" />

oskarth commented on content/es/02_programming-zkps-from-zero-to-hero.md:
```diff
@@ -0,0 +1,752 @@
+
+`just verify_proof example1`
+
+![02_example1_verify_proof](https://hackmd.io/_uploads/r1xiLs08xl.png)
```
These image links should link to assets folder - hackmd images might disappear

oskarth commented on content/es/02_programming-zkps-from-zero-to-hero.md:
```diff
@@ -0,0 +1,752 @@
+## Agradecimientos
+Gracias a Hanno Cornelius, Marc Köhlbrugge, Michelle Lai, lenilsonjr y Chih-Cheng Liang por leer los borradores y brindar retroalimentación sobre esto.
+
+
```
Can add acknowledgement to translators here too as a new paragraph :)

oskarth reviewed (commented):
Awesome work and congrats on your first PR!

Some nitpicks re formatting, then we can merge :)

Gelois0 committed:
Fix Spanish translation: slug prefix, credits, and image links

1. ~~Slug should be without `/es/` prefix - already in folder layout~~ ✅ Removed
2. ~~Added short italics line for translator credits~~ ✅ Done
3. ~~Image links should point to assets folder - hackmd images might disappear~~ ✅ Fixed
4. ~~Added acknowledgement to translators as new paragraph~~ ✅ Done

Gelois0 commented:
I've addressed all the feedback:

1. ✅ Removed `/es/` prefix from slug 
2. ✅ Added translator credits in italics
3. ✅ Fixed image links to point to assets folder 
4. ✅ Added acknowledgement section for translators

Ready for second review when you have a chance. Thanks!

oskarth reviewed (approved):
Awesome! LGTM, thank you :)
